### PR TITLE
editor feature

### PR DESCRIPTION
**Pull request by roquej**

### CODE REVIEW

**Review by ihabzee — APPROVED** [review]
everything seems to be fine—just minor improvements.

File: pages/editProject.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 namespace Stanford\Duster;
 /** @var $module Duster */
```
[the remark this answers is not in the document]
Comment (roquej) [reply pages/editProject.php:3]: Will resolve in a new PR

File: classes/DusterConfigClass.php [hunk 2]
```diff
@@ -30,17 +30,9 @@ public function loadConfig()
...
-
-    public function loadDesignConfig () {
-      // build and send GET request to config webservice
-      $config_url = $this->module->getSystemSetting("starrapi-config-url");
-      // add a '/' at the end of the url if it's not there
-      $config_url = $config_url .
-        ((substr($config_url, -1) === '/') ? "" : "/") . 'design/' . SERVER_NAME . '/' . $this->project_id
-        . '?redcap_user=' . $this->module->getUser()->getUserName();
-      $this->design_config = $this->module->starrApiGetRequest($config_url, 'ddp');
+        $config_object = $this->module->starrApiGetRequest($config_url, 'ddp');
```
[the remark this answers is not in the document]
Comment (roquej) [reply classes/DusterConfigClass.php:33]: Will resolve in a new PR

File: pages/editProject.php [hunk 3]
```diff
@@ -12,25 +13,24 @@
 $pid = $module->getProjectId();
 $project_status = $module->getProjectStatus($pid);
 $user_rights = $module->getUser()->getRights($pid);
+$irb = $module->getProjectIrb($pid);
```
[the remark this answers is not in the document]
Comment (roquej) [reply pages/editProject.php:16]: Will resolve in a new PR

**Pull request merged**